exo-go/test_helpers/clean_feature_context.go

```diff
@@ -31,7 +31,10 @@ func startAndRemoveContainer(dockerClient *client.Client, imageName string) erro
 	if err != nil {
 		return err
 	}
-	return dockerClient.ContainerRemove(ctx, createdBody.ID, types.ContainerRemoveOptions{})
+	if err := dockerClient.ContainerStart(ctx, createdBody.ID, types.ContainerStartOptions{}); err != nil {
```

Comment:
Nevermind, let's go with the steps we discussed over Slack for creating dangling images and volumes.